ios/RN/RNCamera.m

```diff
@@ -167,19 +166,17 @@ - (void)willMoveToSuperview:(nullable UIView *)newSuperview;
              name:UIApplicationDidChangeStatusBarOrientationNotification
            object:nil];
 
-        [[NSNotificationCenter defaultCenter] addObserver:self
-                 selector:@selector(bridgeDidBackground:)
-                     name:UIApplicationDidEnterBackgroundNotification
-                   object:nil];
+        [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(sessionWasInterrupted:) name:AVCaptureSessionWasInterruptedNotification object:self.session];
```

Comment:
will this happen only when audio is enabled?

Reply:
This fires when anything causes the session to be interrupted. Not only audio, but also the app going to background. It basically replaces the background/foreground events that were previously manually stopping the session, and which was wrong (the session is automatically stopped/resumed when the app goes to background/foreground).

If our session was never set to use audio, audio interruptions should not interrupt the session; hence, this event will not fire in that case.

Lastly, there were cases that our session could be interrupted, even if the app did not go to background. A good example of this would be during a phone call. When a phone call is taking place, opening the app drawer will also pause the app (even if it is not sent to background), interrupting any recording. When this happens, the previous code would not properly set the video interrupted flag.